### ISSUE TITLE
Kibana keeps logging out when try to add data

### ISSUE THREAD

**Issue by SulimanLab** [bitnami/charts#4670]
Kibana keeps logging out when try to add data, someone have mentioned that it could be because session expiry and it has to do with "sessionAffinity" need to be "ClientIP" but you guys does not support it !

**Comment by dani8art** [comment]
Hi @SulimanLab, could you please elaborate more? I will need your `values.yaml` or command to try to reproduce the issue.

As far I understand, you are deploying a kibana chart using several replicas so your service is not keeping the session up, right?

**Comment by elvsp1n** [comment]
Hi Dani8art,
    I am not sure if its related by after trying to install the helm with kibana, kibana timesout while trying to connect to es. Es seems to be fine, we are able to use it and even connecting to kibana and running from the kibana container return a reply from es as you can see here:

```
I have no name!@es-kibana-5b54c86b7-kvjd8:/$ curl http://es-coordinating-only:9200
{
  "name" : "es-coordinating-only-66c654d747-z4t4p",
  "cluster_name" : "elastic",
  "cluster_uuid" : "BIl83ROfSuKOaQkmgECp3Q",
  "version" : {
    "number" : "7.10.0",
    "build_flavor" : "oss",
    "build_type" : "tar",
    "build_hash" : "51e9d6f22758d0374a0f3f5c6e8f3a7997850f96",
    "build_date" : "2020-11-09T21:30:33.964949Z",
    "build_snapshot" : false,
    "lucene_version" : "8.7.0",
    "minimum_wire_compatibility_version" : "6.8.0",
    "minimum_index_compatibility_version" : "6.0.0-beta1"
  },
  "tagline" : "You Know, for Search"
}
I have no name!@es-kibana-5b54c86b7-kvjd8:/$ 

service"],"pid":26,"message":"Waiting until all Elasticsearch nodes are compatible with Kibana before starting saved objects migrations..."}
{"type":"log","@timestamp":"2020-12-11T03:29:00Z","tags":["info","savedobjects-service"],"pid":26,"message":"Starting saved objects migrations"}
{"type":"log","@timestamp":"2020-12-11T03:29:00Z","tags":["info","savedobjects-service"],"pid":26,"message":"Creating index .kibana_1."}
{"type":"log","@timestamp":"2020-12-11T03:29:30Z","tags":["error","elasticsearch","data"],"pid":26,"message":"[TimeoutError]: Request timed out"}
{"type":"log","@timestamp":"2020-12-11T03:29:30Z","tags":["warning","savedobjects-service"],"pid":26,"message":"Unable to connect to Elasticsearch. Error: Request timed out"}
{"type":"log","@timestamp":"2020-12-11T03:30:02Z","tags":["error","elasticsearch","data"],"pid":26,"message":"[TimeoutError]: Request timed out"}
{"type":"log","@timestamp":"2020-12-11T03:30:35Z","tags":["error","elasticsearch","data"],"pid":26,"message":"[TimeoutError]: Request timed out"}
{"type":"log","@timestamp":"2020-12-11T03:31:07Z","tags":["error","elasticsearch","data"],"pid":26,"message":"[TimeoutError]: Request timed out"}
{"type":"log","@timestamp":"2020-12-11T03:31:40Z","tags":["error","elasticsearch","data"],"pid":26,"message":"[TimeoutError]: Request timed out"}
{"type":"log","@timestamp":"2020-12-11T03:31:46Z","tags":["info","plugins-system"],"pid":26,"message":"Stopping all plugins."}

```
Any idea why this could be happening? I installed it with the following command:
helm install es --set global.kibanaEnabled=true bitnami/elasticsearch

**Comment by dani8art** [comment]
Hi @elvsp1n,

I was not able to reproduce your issue, I've just executed your command, and all is working as expected. 

Are you installing the chart from a very fresh instance, there could be any PVC from a previous deployment? if so, you would need to clean them, please try to remove them and install the chart again.

```console
$ k delete pvc -l "app.kubernetes.io/instance=es"
```

BTW, I think this couldn't be related to the other issue, please, try this and if it is not solved open a new ticket to better handle this one, thanks!
